javatests/arcs/android/systemhealth/testapp/TestEntity.kt

```diff
@@ -28,18 +28,21 @@ class TestEntity(
     text: String = "",
     number: Double = 0.0,
     boolean: Boolean = false,
-    val reference: Reference? = null,
+    inlineText: String = "",
+    var reference: Reference? = null,
```

Comment:
you should consider generating TestEntity from a .arcs manifest instead of building it manually here

Reply:
There is a plan to adapt the whole app to using Arcs recipes and particles.
Will remove entire entity defs by that tim.